ghedesigner/tests/test_load_aggregation.py

```diff
@@ -0,0 +1,267 @@
+from ghedesigner.borehole import GHEBorehole
+from ghedesigner.borehole_heat_exchangers import SingleUTube, MultipleUTube, CoaxialPipe
+from ghedesigner.coordinates import rectangle
+from ghedesigner.enums import BHPipeType, TimestepType
+from ghedesigner.gfunction import calc_g_func_for_multiple_lengths
+from ghedesigner.ground_heat_exchangers import GHE
+from ghedesigner.media import Pipe, Soil, Grout, GHEFluid
+from ghedesigner.simulation import SimulationParameters
+from ghedesigner.tests.ghe_base_case import GHEBaseTest
+from ghedesigner.utilities import eskilson_log_times
+
+
+class TestLoadAggregation(GHEBaseTest):
```

Comment:
It would be much preferred if you could update this test to follow this example.

https://github.com/BETSRG/GHEDesigner/blob/e055caa449285499a58a043961ee27946dcd3ca0/ghedesigner/tests/test_find_design_rectangle.py#L6-L26

The previous tests were quite verbose. The new approach is a much more compact way to do the same thing.

Reply:
Thanks! I fused the two concepts and made the test even more concise.